INSTRUCTIONS.md

```diff
@@ -6,11 +6,9 @@ Throughout, the main things being assessed are:
 
 * Design        e.g. variable names, code readability, having a logical flow and process, labeling of graphs
 * Data quality  e.g. paying attention to outliers, missing data etc.
-* Explanations  e.g. justifying your approach, comments in the code file will be sufficient
-  
-We are looking for a well-commented code file, and requested outputs saved in `output/`. 
+* Explanations  e.g. justifying your approach
 
-Of course if you find it easier to use R Markdown to provide explanations/justifications in a report format that will be more than acceptable. 
+We are looking for a well-commented R script or R Markdown script, and for outputs (charts etc.) saved in `output/`. 
```

Comment:
Pedantic point: should:
and for outputs (charts etc.) saved in `output/`

read as:
and for outputs (charts etc.) **to be** saved in `output/`

Reply:
Good catch, thanks.